docs/tools/vdb_table/data/nucliadb.json

```diff
@@ -110,43 +110,43 @@
   },
   "pricing": {
     "value": "",
-    "source_url": "",
-    "comment": ""
+    "source_url": "https://nuclia.com/pricing/",
+    "comment": "Pricing|Nuclia"
   },
   "in_process": {
     "support": "none",
     "source_url": "",
     "comment": ""
   },
   "multi_tenancy": {
-    "support": "",
+    "support": "full",
     "source_url": "",
     "comment": ""
   },
   "disk_index": {
-    "support": "",
+    "support": "full",
     "source_url": "",
     "comment": ""
   },
   "ephemeral": {
-    "support": "",
+    "support": "none",
     "source_url": "",
     "comment": ""
   },
   "sharding": {
-    "support": "",
+    "support": "full",
     "source_url": "",
     "comment": ""
   },
   "doc_size": {
     "bytes": 0,
-    "unlimited": false,
+    "unlimited": true,
     "source_url": "",
-    "comment": ""
+    "comment": "Max vectors of 50K"
```

Comment:
is this referring to number of vectors in an index? or something else?

Reply:
More detail explanation has been provided